src/us/kbase/userandjobstate/UserAndJobStateServer.java

```diff
@@ -579,14 +580,19 @@ private URL getURL(final Map<String, String> config, final String param) {
 	private ConfigurableAuthService setUpAuthClient(
 			final Map<String, String> config) {
 		final URL authURL = getURL(config, KBASE_AUTH_URL);
+		final String authAllowInsecure = config.get(INSECURE_AUTH_URL);
 		final URL globusURL = getURL(config, GLOBUS_AUTH_URL);
 		if (authURL == null || globusURL == null) { // a url was invalid
 			return null;
 		}
+		System.out.println(UserAndJobStateServer.class.getName() + ": " + INSECURE_AUTH_URL +" = " + 
```

Comment:
This should be logged the same way as the other params in the constructor.

Reply:
I basically cloned this code from the NMS repo, I actually had some code that displayed it along with the rest of the config params the way it was in another repo, but didn't push it  since it seemed redundant, I'll put that code back in.